webclient/src/services/DexieService.ts

```diff
@@ -18,7 +20,11 @@ class DexieService {
   private db: Dexie = new Dexie('Webatrice');
 
   constructor() {
-    this.db.version(1).stores(StoreKeyIndexes);
+    this.db.version(2).stores(StoreKeyIndexes);
```

Comment:
Do we include an upgrade path for users who may have an outdated local?

Reply:
Not yet.

Reply:
Should ticket this for later